pkg/executables/helm.go

```diff
@@ -235,9 +215,9 @@ func (h *Helm) UpgradeChartWithValuesFile(ctx context.Context, chart, ociURI, ve
 		"--wait",
 	}
 	for _, opt := range opts {
-		opt(h)
+		opt(h.helmOptions)
```

Comment:
wow is this updating the receiver? it seems like this is how it used to be before, which is not great. Do you mind creating a ticket or something to track this and come back and change it? this is not thread safe

Reply:
Yeah, I'll create the ticket and address it right after

Reply:
Done

https://github.com/aws/eks-anywhere/issues/7176